clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -298,8 +289,8 @@ private void onFailure(final long currentTimeMs,
         }
 
         private void retry(final long currentTimeMs) {
-            onFailedAttempt(currentTimeMs);
-            onSendAttempt(currentTimeMs);
+            this.onFailedAttempt(currentTimeMs);
```

Comment:
currentTimeMs is when the request is sent, not when the response is receive. Could we pass along `ClientResponse.receivedTimeMs`?

Reply:
Thanks for catching this.

Reply:
Addressed, see line #259